data/tokens.json

```diff
@@ -1499,30 +1499,6 @@
       }
     }
```

Comment:
@tsnikolov I think we should define tokens for paddings and margins(gap) on the main components and it's parts.